packages/turf-isolines/index.js

```diff
@@ -37,7 +33,7 @@ var featureCollection = helpers.featureCollection;
  * //addToMap
  * var addToMap = [isolines];
  */
-module.exports = function (pointGrid, breaks, zProperty, propertiesToAllIsolines, propertiesPerIsoline) {
+export default function (pointGrid, breaks, zProperty, propertiesToAllIsolines, propertiesPerIsoline) {
```

Comment:
@stebogit For some reason the test outputs fixtures had to change..? Don't know why this would of caused it by just switching over to ES6 modules.

![image](https://user-images.githubusercontent.com/550895/30776046-2fd62394-a06d-11e7-985f-06ac97ffb731.png)

Reply:
No idea... 🤔